.stylelintrc.js

```diff
@@ -0,0 +1,99 @@
+module.exports = {
+    'defaultSeverity': 'warning',
```

Comment:
I'm okay with this line being here for now, but I've added an AC for it to be removed as part of https://github.com/silverstripe/.github/issues/228